src/com/javarush/task/task08/task0815/Solution.java

```diff
@@ -0,0 +1,83 @@
+package com.javarush.task.task08.task0815;
+
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.Map;
+
+/*
+Перепись населения
+*/
+
+public class Solution {
+    public static Map<String, String> createMap() {
+        //напишите тут ваш код
+        Map<String, String> map = new HashMap<>();
+        map.put("Фамилия", "Имя");
+        map.put("Иванов", "Иван");
+        map.put("Петров", "Петр");
+        map.put("Сидоров", "Сидр");
+        map.put("Пыл", "Саша");
+        map.put("Лещенко", "Паша");
+        map.put("Фамилия1", "Имя1");
+        map.put("Фамилия2", "Имя2");
+        map.put("Фамилия3", "Имя3");
+        map.put("Фамилия4", "Имя4");
+        return map;
+
+    }
+
+    public static int getCountTheSameFirstName(Map<String, String> map, String name) {
+        //напишите тут ваш код
+        Iterator iterator = map.entrySet().iterator();
+        int count = 0;
+        while (iterator.hasNext()) {
+            Map.Entry entry = (Map.Entry) iterator.next();
+            if (entry.getValue().equals(name)) {
+                count++;
+            }
+
+
+        }
+        return count;
+    }
+
+    public static int getCountTheSameLastName(Map<String, String> map, String lastName) {
+        //напишите тут ваш код
+        Iterator iterator = map.entrySet().iterator();
+        int count = 0;
+        while (iterator.hasNext()) {
+            Map.Entry entry = (Map.Entry) iterator.next();
+            if (entry.getKey().equals(lastName)) {
+                count++;
+            }
+
+
+        }
+        return count;
```

Comment:
Вариант использования форич с мапой (можно доставать отдельно ключ или значение или и то и другое)
```suggestion
        int count = 0;
        for (String keys : map.keySet()) {
            if (keys == lastName || keys.equals(lastName))
                count++;
        }
        return count;
```